docs/quick-start/getting-started.md

```diff
@@ -0,0 +1,19 @@
+# Getting Started with Ignite and React Native
+
+Welcome to Ignite, the ideal starting app for React Native, best practices, generators, and more. This page will walk you through installing Ignite and using Ignite to generate a React Native Project.
+
+To use Ignite, you will need to install React Native. The best way to install React Native is to follow the React Native installation instructions found on official React Native website, [here](http://facebook.github.io/react-native/docs/getting-started.html). These instructions will walk your through installing React Native on your specific platform.
+
+## Installing Ignite Dependencies
+
+Installing Ignite is easy with one simple terminal command:
+
+```
+npm install -g ignite
```

Comment:
It's `react-native-ignite`. We have the wrong package name in the source code! Ha!  Il. Fix that.